src/test/java/org/terasology/flexiblepathfinding/helpers/MapWorldProvider.java

```diff
@@ -171,21 +173,41 @@ public boolean isBlockRelevant(Vector3i pos) {
         return false;
     }
 
+    @Override
+    public boolean isBlockRelevant(Vector3ic pos) {
+        return false;
+    }
+
     @Override
     public boolean isBlockRelevant(Vector3f pos) {
         return false;
     }
```

Comment:
You can remove the non read-only equivalents, they're obsolete now.

Reply:
that's the teramath vector. I think i need it from the interface and I believe its marked as depreciated.